Project_JS/scripts/calculator.js

```
@@ -0,0 +1,29 @@
+
+function deleteLast() {
+    if (display.innerText.length === 1) {
+        display.innerText = '0';
+    } else {
+        display.innerText = display.innerText.slice(0, -1);
+    }
+}
+
+function calculate() {
+    try {
+        display.innerText = eval(display.innerText.replace(/×/g, '*'));
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)